src/proto/beats.proto

```diff
@@ -10,22 +11,73 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
+message BpmMarker {
+  optional int32 beat_index = 1;
+  optional double bpm = 2;
```

Comment:
Here we mixxx time based info with smaples based info in one data structure. 
We should not do this. Instead we should give here the beat length in samples. This is what we need anyway when accessing the data. 
And that is the output of the analyzer.

Reply:
I have just skimmed through the code and it turns out that we can remove sample rated dependency in various places. 
This avoids also rounding issues. Let's store the data in the format it is detected and needed later. No conversion to human readable format. 

Adding a sample rate field to the protobuffer is however a good idea to be able to create the time based representation at any time.